src/ip/Intercept.cc

```diff
@@ -364,21 +364,21 @@ Ip::Intercept::PfInterception(const Comm::ConnectionPointer &newConn, int silent
         int xerrno = errno;
         if (xerrno != ENOENT) {
             if (!silent) {
-                debugs(89, DBG_IMPORTANT, HERE << "PF lookup failed: ioctl(DIOCNATLOOK): " << xstrerr(xerrno));
+                debugs(89, DBG_IMPORTANT, "PF lookup failed: ioctl(DIOCNATLOOK): " << xstrerr(xerrno));
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.